src/rust/engine/async_semaphore/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+version = "0.0.1"
+name = "async_semaphore"
+authors = [ "Pants Build <pantsbuild@gmail.com>" ]
+
+[dependencies]
+futures = "0.1"
```

Comment:
is the version regression from `0.1.16` -> `0.1` intended here? weren't there changes that were upstreamed in later versions of `futures` that we relied on?